src/ONE.Models.CSharp/Extensions/EnumRelativeDateRangeExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿using ONE.Models.CSharp.Imposed.Enums;
+using System;
+
+namespace ONE.Models.CSharp.Extensions;
+
+public static class EnumRelativeDateRangeExtensions
+{
+    public static string ToI18NKey(this EnumRelativeDateRange dateRange)
```

Comment:
This seems like maybe not the right place for this code.  We are coupling our models package to our I18NKeys? If this is needed by Enterprise.Data it should go in Enterprise.Data repository.  Test.Web should probably have it's own copy so that we are not using the code to test the code.  If you really want this to be shared it could go in the helpers package.